content/ui-patterns/focus-management.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Focus management
+---
+
+## Overview
+
+Managing focus within a page or application is essential for users to successfully navigate, complete actions, and understand where they are within a web page or application. A focus indicator is usually seen visually as a blue outline around an interactive element when that element has focus.
+
+## Why?
+
+Not all users are able to use a mouse to navigate a web page or application. Keyboard-only and screen reader users rely on navigating and using a web page with only a keyboard, so they will be unable to perform certain actions if they cannot navigate to all elements.
+
+To meet [WCAG Guideline 2.1 (Keyboard Accessible)](https://www.w3.org/TR/WCAG21/#keyboard-accessible), all interactive elements need to be accessed and activated by a keyboard alone.
+
+## How to test focus management
+
+To verify proper focus management, try using the feature or page with only a keyboard. Use the Tab key to navigate forward and Shift + Tab to move backwards.
+
+Things to keep in mind:
+
+- You can navigate to every interactive element (i.e. buttons, form fields, links)
+- You see a visual focus indicator for every element (i.e. a blue outline)
+- The tabbing order is logical based on the page structure and content
+- You are not trapped anywhere, and focus is never lost (i.e. when deleting a comment, focus is returned to a logical place)
+- If you can trigger a new element (i.e. a modal), focus is moved to that modal when opened, and returned to the page when closed
+
+## For designers
```

Comment:
I'm thinking the "For designers" and "For engineers" headings should actually be h3 and there should be a h2 heading above them with a short intro of _what_ is for designers? Are the guidelines? Something else?